.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature Request] Title"
+labels: ''
+assignees: ''
+
+---
+
+## Is your feature request related to an existing problem?  Please describe.
```

Comment:
I would open with a "Why is this important/What are the usecases"

Reply:
Amended